uber/templates/regform.html

```diff
@@ -258,7 +258,12 @@
     {% if c.COLLECT_EXACT_BIRTHDATE %}
         <label for="birthdate" class="col-sm-2 control-label">Date of Birth</label>
         <div class="col-sm-6">
-            <input type='text' class="form-control date" name="birthdate" value="{{ attendee.birthdate|date:"Y-m-d"}} "/>
+            <nobr>
+                <input type='text' class="form-control date" name="birthdate" value="{{ attendee.birthdate|date:"Y-m-d"}} "/>
+                {% if c.PAGE_PATH == '/registration/form' and attendee.birthdate %}
+                    ({{ attendee.age_group_conf.desc }})
```

Comment:
Yes, sorry I forgot to mention it, but I'm gonna do that as part of a separate PR, since I need to make some performance improvements to the check-in form tonight anyway.